### PR TITLE
New package : edk2-shell-202005

### PR DESCRIPTION
**Pull request by pandom79**

### CODE REVIEW

**Comment by pandom79** [comment]
Hi, 
I'm porting this package from [here](https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/edk2).
In the pkgbuild as architecture there is "any" but inside it is built with ia32 and x64.
How to proceed?

File: srcpkgs/edk2-shell/template [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+revision=1
+wrksrc=edk2-edk2-stable202005
+hostmakedepends="nasm"
+makedepends="libuuid-devel python3"
+short_desc="EDK2 UEFI Shell"
+maintainer="Domenico Panella <pandom79@gmail.com>"
+license="BSD-2-Clause-Patent"
+homepage="https://github.com/tianocore/edk2"
+distfiles=" https://github.com/tianocore/edk2/archive/edk2-stable${version}.tar.gz
+			https://github.com/google/brotli/archive/v${_brotli_ver}.tar.gz"
```
Comment (sgn) [inline srcpkgs/edk2-shell/template:14]: We ships `brotli`. Does it work with system brotli?

Comment (pandom79) [reply srcpkgs/edk2-shell/template:14]: If you see the PKGBUILD, two packages are created (edk2-shell and edk2-ovmf). How do I merge into voidlinux? edk2-shell is not a subpackage of edk2-ovmf.
Another question: I have not seen that a pull request [17225](https://github.com/void-linux/void-packages/pull/17225) already exists that deals with this functionality.
What do we do? it looks old ......

**Comment by sgn** [comment]
I'm in favour of that one. Closing as duplicate of #17225

**Pull request closed without merging**